src/category_theory/limits/shapes/pullbacks.lean

```diff
@@ -503,6 +503,25 @@ def pullback_cone.of_cone
 { X := t.X,
   π := t.π ≫ (diagram_iso_cospan F).hom }
 
+/--
+A diagram `walking_cospan ⥤ C` is isomorphic to some `pullback_cone.mk` after composing with
+`diagram_iso_cospan`.
+-/
+def pullback_cone.iso_mk {F : walking_cospan ⥤ C} (t : cone F) :
```

Comment:
Could you add a `@[simps]`? (You can use `@[simps?]` first to check what lemmas are generated to make sure they look reasonable, before changing to `@[simps]`.)

Similarly for thee pushout case.